main.go

```diff
@@ -69,6 +69,7 @@ LEARN MORE:
 		lambda(),
 		version(),
 		calls(),
+		deploy(),
```

Comment:
This is already there: https://github.com/fnproject/cli/pull/120/files#diff-7ddfb3e035b42cd70649cc33393fe32cR77

Reply:
Oh, I see, it's not showing --help.